src/content/snippets/styled-components/Page.js.mdx

```diff
@@ -0,0 +1,86 @@
+```js
+// ./src/stories/Page.js
+
+import React from 'react';
+import styled from 'styled-components';
+
+import { Header } from './Header';
+import './page.css';
```

Comment:
Still used?

Reply:
Yes, just to avoid needing to rewrite the entire component with the same styles